Source/Urho3D/Graphics/AnimationTrack.cpp

```diff
@@ -223,4 +308,67 @@ VariantType VariantAnimationTrack::GetType() const
     return keyFrames_.empty() ? VAR_NONE : keyFrames_[0].value_.GetType();
 }
 
+bool VariantAnimationTrack::Serialize(Archive& archive)
+{
+    SerializeValue(archive, "name", name_);
+    if (archive.IsInput())
+        nameHash_ = name_;
+    SerializeValue(archive, "baseValue", baseValue_);
+    SerializeEnum(archive, "interpolation", interpMethodNames, interpolation_);
+    SerializeValue(archive, "splineTension", splineTension_);
+    if ((interpolation_ == KeyFrameInterpolation::TensionSpline ||
```

Comment:
Tension spline should generate tangents instead of serialization.
`Commit` should be called on load.